modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -188,6 +188,7 @@ enum VideoCaptureProperties {
        CAP_PROP_HW_ACCELERATION_USE_OPENCL=52, //!< (**open-only**) If non-zero, create new OpenCL context and bind it to current thread. The OpenCL context created with Video Acceleration context attached it (if not attached yet) for optimized GPU data copy between HW accelerated decoder and cv::UMat.
        CAP_PROP_OPEN_TIMEOUT_MSEC=53, //!< (**open-only**) timeout in milliseconds for opening a video capture (applicable for FFmpeg back-end only)
        CAP_PROP_READ_TIMEOUT_MSEC=54, //!< (**open-only**) timeout in milliseconds for reading from a video capture (applicable for FFmpeg back-end only)
+       CAP_PROP_STREAM_OPEN_TIME =55, //<! (read-only) time in seconds since Jan 1 1970 when stream was opened. Applicable for FFmpeg backend only. Useful for RTSP and other live streams
```

Comment:
> in seconds

Lets use nanoseconds to keep this value accurate and avoid division (FFmpeg uses microseconds internally)

Reply:
Ok. I changed the name of the cap prop (added _NSEC to the end) to reflect the units

Reply:
I looked on that once more.

`int64_t` is good for nanoseconds, but `double` which is used to return property values - doesn't. Double's 53 mantissa bits are reached in 104 days.

Details about similar problem: https://groups.google.com/g/python-ideas/c/RLRBS15Gofc

Seconds / milliseconds is not enough accurate for sensors (without fractional part).
Perhaps, microseconds (`_USEC`) should work.

Reply:
Using seconds/milliseconds with the fractional part would necessitate a division (is there a compiler macro #ifdef-type thing to allow that?), so I don't think we'd want to go with that. Microseconds fits in a double until ~2170 (log2  [86400*1e6 * 365 * 200] is less than 53). I'm going to change to microseconds